pilot/docker/mixer_bootstrap.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+eval "cat <<EOF
+$(</etc/istio/proxy/envoy_istio_policy.json)
+EOF
+" > /etc/istio/proxy/envoy_istio_policy.json
+
+eval "cat <<EOF
+$(</etc/istio/proxy/envoy_istio_policy_auth.json)
+EOF
+" > /etc/istio/proxy/envoy_istio_policy_auth.json
+
+eval "cat <<EOF
+$(</etc/istio/proxy/envoy_istio_telemetry.json)
+EOF
+" > /etc/istio/proxy/envoy_istio_telemetry.json
+
+eval "cat <<EOF
+$(</etc/istio/proxy/envoy_istio_telemetry_auth.json)
+EOF
+" > /etc/istio/proxy/envoy_istio_telemetry_auth.json
+
+# PILOT-AGENT
+/usr/local/bin/pilot-agent $@
```

Comment:
I hate to see cat/seds again... I think we can fork pilot-agent to do the combination of templating + startup instead of doing templating first, then start-up. Alternatively, we could just embed these YAMLs in helm charts. Lots of ideas to get rid of duplication.

Reply:
Agree that this is ugly. I wanted to keep things as limited in scope as possible for the initial PR.  Are you suggesting we make the start-up scripts part of the deployment (as distinct from part of the proxy image)?

Would you be opposed to optimizing this in a follow-on PR, once initial functionality is established? As we will, at some point, need to convert to v2 config, perhaps we can clean this up as part of that?

Reply:
Yes, I do not intent to hold this PR over this. You're doing the hard work of making it work and testing, the clean-up should be an easy follow-up.